grafanalib/core.py

```diff
@@ -98,7 +98,8 @@ def to_json_data(self):
 DURATION_FORMAT = "dtdurations"
 NO_FORMAT = "none"
 OPS_FORMAT = "ops"
-PERCENT_UNIT_FORMAT = "percentunit"
+PERCENT_FORMAT = "percent" # 1%, 80%, etc
+PERCENT_UNIT_FORMAT = "percentunit" # 0.01, 0.8
```

Comment:
while we're here - are there any other units in grafana we are missing?  This is a long long list that might be worth picking over: https://github.com/grafana/grafana/blob/bcf88f61575dc1bae2c67e6a306c45b52648f7c7/packages/grafana-ui/src/utils/valueFormats/categories.ts#L23